ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml

```diff
@@ -1404,7 +1406,7 @@ periodics:
               workflow: ipi-$(CLUSTER_TYPE)-ovn-hybrid-custom-vxlan-port
           - as: launch-techpreview
             steps:
-              cluster_profile: $(CLUSTER_TYPE)
+              cluster_profile: $(CLUSTER_PROFILE)
```

Comment:
thanks @ayesha54!  we will also want to update the cluster_profile in these steps as well:
https://github.com/openshift/release/pull/23576/files#diff-37659dcced1f233fd38cb7c243a2bd003b6d0dc7c31fcca5be2ad08319d56fe0L1381-L1403 .